dashboard/test/helpers/delete_accounts_helper_test.rb

```diff
@@ -974,13 +974,24 @@ def run(*_args, &_block)
 
   test "clears form_data from pd_facilitator_program_registrations" do
     teacher = create :teacher
-    registration = create :pd_facilitator_program_registration, user: teacher
-    refute_equal '{}', registration.form_data
+    ActiveRecord::Base.connection.exec_query(
+      <<-SQL
+        INSERT INTO `pd_facilitator_program_registrations` (user_id, form_data, created_at, updated_at)
+        VALUES (#{teacher.id}, '{\"country\": \"USA\"}', '#{Time.now.to_s(:db)}', '#{Time.now.to_s(:db)}')
```

Comment:
wouldn't you want to save Time.now in a variable to make sure you get the same value always for created_at and updated_at?

Reply:
Why do those values need to be the same? I don't see why it matters

Reply:
you are right since it is a test.